terms/empiricalriskminimisation_nf.tex

```diff
@@ -0,0 +1,51 @@
+
+\section{Loss Functions}
+A simple supervised learning scenario comprises of the following. Typically there would be one or more inputs or features denoted as $x$ and the target output or labels denoted as $y$.
+Using the combination of features and output $(x,y)$, a prediction function $f(x)$ is built where, given a new input $x'$ gives the best estimate for the value of $y'$, assuming that the data follows an underlying distribution that is not known. Once a set of predictions is generated, a way of measuring how good these predictions are is required. For a single data point this is simply the distance between $y$ and $y'$. Repeating this process for multiple points will give us a loss function \citep{Wehenkel2018}:
+\begin{equation}
+    L(f(x),y)
+\end{equation}
+The loss function gives us the cost our prediction function (f(x)) which gives a clear indication of how efficient the function is at predicting the expected output. Common loss functions are squared-error loss  and mean square error. This calculation of "cost" of the function can give us an indication of the risk of the prediction function. Therefore, the goal of finding the most optimal prediction function turns out to be the goal of finding the function which minimises the true risk.
+
+\section{True Risk}
+
+True Risk is the prediction error on the entire population of data. In almost all cases, it is impossible to collect the entire data as it almost impossible. For example, in the case of predicting the gender of a person based on their height and weight, every single person across the globe would need to have their height and weight collected all into one set of data. Since entire population is not available there needs to be a way of minimising the true risk, without the true distribution being known. 
+
+\section{Empirical Risk Minimisation}
+This is where Empirical Risk Minimisation comes in. When constructing the supervised learning model, the predictive function chosen is the one with the  least discrepancy between predicted values and actual values. In other words, the one with the least risk. This risk is Empirical Risk and the process of finding the function that minimises the risk is Empirical Risk Minimisation. Note the risk is empirical risk and not the true risk as the data is only subset of the entire population. In an ideal scenario the end goal would be to minimise the true risk as much as possible. Due to the fact that the data for calculating the true risk is unavailable this cannot be achieved, however it is said that the empirical risk is almost identical to true risk. Therefore by minimising the empirical risk the true risk should be minimised also.
+
+\section{Equation of Empirical Risk Minimisation}
+
+The following is the equation for the Empirical Risk Minimisation as proposed by \citet{Vapnik}\citep{principles}:
```

Comment:
WHy have you cited vapnik twice here???  Look at the pdf -- it looks bad.  You either refer to the author in text (citet) or reference normally citep.